onadata/libs/serializers/metadata_serializer.py

```diff
@@ -173,6 +173,8 @@ def validate(self, attrs):
 
         if data_file:
             allowed_types = settings.SUPPORTED_MEDIA_UPLOAD_TYPES
+            # add geojson mimetype
+            mimetypes.add_type('application/geo+json', '.geojson')
```

Comment:
Why do we need to do this ? I've removed this line and the tests still pass. Isn't the change in the settings enough ?